server/src/main/java/com/skillmagnet/User/UserController.java

```diff
@@ -32,4 +55,47 @@ public ResponseEntity<User> getUser(@RequestParam("username") String username) {
         return new ResponseEntity<>(user, HttpStatus.OK);
```

Comment:
I see why you added this back but imo probably want the client to not use this. We can keep for now but maybe should tell FE people to not use going forward